stdlib/public/core/GroupInfo.json

```diff
@@ -233,7 +233,11 @@
     "Codable.swift",
     "LegacyABI.swift",
     "MigrationSupport.swift",
-    "PtrAuth.swift"
+    "PtrAuth.swift",
+    "DoubleWide.swift",
+    "Duration.swift",
+    "DurationProtocol.swift",
+    "Instant.swift"
```

Comment:
Nit: I think it would make sense to create a new group for these, called "Clock" (or something similar).